hooks/post_gen_project.py

```diff
@@ -287,6 +298,67 @@ def set_flags_in_settings_files():
     set_django_secret_key(os.path.join("config", "settings", "test.py"))
 
 
+def set_bootstrap_css_link(file_path, version=None, sri=None):
+    tag = (
+        '<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/'
+        '{}/css/bootstrap.min.css" integrity="{}" crossorigin="anonymous" '
+        'referrerpolicy="no-referrer" /> '
+    ).format(version, sri)
+    bootstrap_css_tag = set_flag(file_path, "!!!SET BOOTSTRAP CSS LINK!!!", value=tag)
+    return bootstrap_css_tag
+
+
+def set_bootstrap_js_link(file_path, version=None, sri=None):
+    tag = (
+        '<script defer src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/'
+        '{}/js/bootstrap.min.js" integrity="{}" crossorigin="anonymous" '
+        'referrerpolicy="no-referrer"></script>'
+    ).format(version, sri)
+    bootstrap_js_tag = set_flag(file_path, "!!!SET BOOTSTRAP JS LINK!!!", value=tag)
+    return bootstrap_js_tag
+
+
+def set_bootstrap_version():
+    add_css = "{{ cookiecutter.custom_bootstrap_compilation }}".lower() == "n"
+    add_js = not (
+        "{{ cookiecutter.custom_bootstrap_compilation }}".lower() == "y"
+        and "{{ cookiecutter.js_task_runner }}".lower() == "Gulp"
+    )
+    if not (add_css or add_js):
+        return
+    try:
+        r = urlopen("https://api.cdnjs.com/libraries/bootstrap?fields=versions")
```

Comment:
I'm not sure how I feel about making HTTP requests in the post generation hook, where we don't control the environment very well... Especially since we're still supporting Python 2 (reminds me that it might be time to drop it maybe). Also this might upgrade to a Bootstrap version we don't support yet? e.g. what happens when Bootstrap 6 is out and we're not yet ready?

As a side note, I'm trying to get Dependabot setup on the repo to work for npm, it seems to be working on my fork. 

Once this is done, I'm thinking we could tweak the approach here to :

- have the parts we need from this HTTP call (version & SRI) saved in a JSON file in the repo
- update that JSON file using GH action if a PR is opened/updated and looks like a Dependabot PR for Bootstrap
- use that JSON file in the post gen hook to replace the bits of HTML like you do currently

Not sure if we actually need a JSON file, maybe we could stick to the constants you have now or even leave the script tags in the HTML and replace them. My main idea would be to keep the version from `package.json` and the HTML in line, and update them together in a single PR